src/middlewared/middlewared/plugins/jbof/redfish/target.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import urllib3
+
+from middlewared.service import Service, private
+
+DEFAULT_REDFISH_TIMEOUT_SECS = 10
+HEADER = {'Content-Type': 'application/json', 'Vary': 'accept'}
+REDFISH_ROOT_PATH = '/redfish/v1'
+
+
+class JBOFRedfishService(Service):
+
+    class Config:
+        namespace = 'jbof.redfish'
+        private = True
+
+    @private
```

Comment:
Class is already private, we don't need the private decorators.

Reply:
Thanks, removed.